community/common/src/test/java/org/neo4j/test/Randoms.java

```diff
@@ -89,6 +100,8 @@ public int arrayMaxLength()
     public static final int CSA_LETTERS = CS_LOWERCASE_LETTERS | CS_UPPERCASE_LETTERS;
     public static final int CSA_LETTERS_AND_DIGITS = CSA_LETTERS | CS_DIGITS;
 
+    public static long MAX_NANOS_PER_DAY = 86399999999999L;
```

Comment:
- nextLong already excludes the bound, so no need for a -1 here possibly
- if you for algorithmic reasons require a -1, then it is better to add that in the specific place where it is needed, with a comment as suitable
- you should be able to find a suitable constant already somewhere else or use TimeUnit conversion, i.e. DAYS.toNanos(1)